lib/bundlex/toolchain/common/compilers.ex

```diff
@@ -0,0 +1,36 @@
+defmodule Bundlex.Toolchain.Common.Compilers do
+  defstruct [:c, :cpp]
+
+  def resolve_lang(lang) do
+    cond do
+      lang in ["cpp", "c++", "C++", "CPP", "cplusplus", "CPLUSPLUS"] ->
```

Comment:
I'm definitely against supporting so many possibilities for specifying the same thing. It introduces mess and confusion while providing no real profits. I'd go for supporting `:c` and `:cpp` atoms only.